trusted-types/TrustedTypePolicyFactory-getAttributeType-event-handler-content-attributes.tentative.html

```diff
@@ -20,29 +20,38 @@
       .members.map(member => member.name)
       .filter(name => name.length >= 3 && name.startsWith("on") &&
               !name.startsWith("onwebkit"))
-      .forEach(name => attributeNames.push(name));
+      .forEach(name => attributeNamesWithInterfaceName.push({name, interfaceName}));
   }
 
   const htmlIDL = await (await fetch("/interfaces/html.idl")).text();
-  ["GlobalEventHandlers", "WindowEventHandlers"].forEach(interfaceName => {
+  ["GlobalEventHandlers"].forEach(interfaceName => {
```

Comment:
I think these `WindowEventHandlers` event names correspond to content attributes of the HTML body element: https://html.spec.whatwg.org/#the-body-element so we should keep them.

Reply:
Good catch!